lib/attachinary/utils.rb

```diff
@@ -79,5 +96,10 @@ def self.process_options(options)
       options
     end
 
+    def self.clean_hash(hash)
+      return hash if Rails::VERSION::MAJOR < 5
```

Comment:
This check is not strictly necessary, but safer in case this gem gets bumped before we upgrade to rails 5.

Reply:
If we end up leaving the raise behavior in on rails 4 before going to 5, we're going to have to go back to this.

Reply:
@ebenoist Are you saying we might switch the behavior of rails 4 strong params before shipping rails 5? In that case then yes this switch isn't useful.

Reply:
Yeah. I suspect this will be a phased release.

Step 1: Get Rails 5 green
Step 2: Port as MANY of the Rails 4/5 conditionals to be compatible in both versions
Step 3: ???